### PR TITLE
test: create seedlot registration test and minor fixes

### PR DESCRIPTION
**Pull request by Wennio-Oliveira**
# Description

It was created a flow to cover some scenarios during the Seedlot registration process and did some minor fixes requested by lint

# How Has This Been Tested?

It was tested locally using SparWebApp on test environment

seedlotRegistration03

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-84-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-84-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by iatdaitan — APPROVED** [review]

**Review by RMCampos — COMMENTED** [review]
This new test case is failing for me.

File: frontend/cypress/support/commands.ts [hunk 1]
```diff
@@ -36,10 +36,10 @@ Cypress.Commands.add('login', () => {
   cy.getByDataTest('landing-button__bceid').click();
   cy.get('#bceidLogo').should('be.visible');
   cy.get('input[name=user]')
-        .clear()
-        .type(USERNAME, { delay: 50 });
+    .clear()
+    .type(USERNAME, { delay: 50 });
```
Comment (RMCampos) [inline frontend/cypress/support/commands.ts:40]: Can you please update the `README.md` file with the correct name of the cypress environment variables?

**Review by RMCampos — CHANGES_REQUESTED** [review]

**Review by mgaseta — APPROVED** [review]
Besides, Ricardo's markups, LGTM!

**Review by RMCampos — APPROVED** [review]
LGTM! Good job 😎

**Pull request merged**